pkg/cue/process/handle.go

```diff
@@ -42,6 +42,7 @@ type Context interface {
 	PushData(key string, data interface{})
 	GetCtx() context.Context
 	SetCtx(context.Context)
+	SetCtxWithValue(key, value interface{})
```

Comment:
you should not add a interface for so little convenience. 

It can be done by just one line:
```
ctx.SetCtx(context.WithValue(ctx.GetCtx(), key, value))
```

Reply:
> 
`ctx.SetCtx()` can't set a new context, If we want to set a new context, we may need a `setCtxForce` method. @wonderflow 
https://github.com/kubevela/kubevela/blob/0c190ef32948561b577ccb0ca8432aee74bcfc80/pkg/cue/process/handle.go#L330-L335

Reply:
Let's check why it's can't be set a new one. \cc @leejanee

Reply:
You can change that function to set a new context even there exists one

Reply:
ok, I will change it